plugins/mac-avcapture/av-capture.mm

```diff
@@ -604,7 +604,7 @@ static inline bool update_audio(obs_source_audio *audio,
 {
 	size_t requiredSize;
 	OSStatus status = CMSampleBufferGetAudioBufferListWithRetainedBlockBuffer(
-		sample_buffer, &requiredSize, nullptr, NULL, nullptr, nullptr,
+		sample_buffer, &requiredSize, nullptr, 0, nullptr, nullptr,
```

Comment:
I don't get a warning about this on Xcode - is this emitted by clang via Ninja?

Reply:
While I can't answer the question in particular, the argument is a `size_t` so passing `NULL` there was wrong in the first place.

Reply:
CI, so Ninja indeed